src/index.ts

```diff
@@ -48,14 +54,16 @@ async function nitroModule(nitro: Nitro) {
     }
   }
 
-  consola.box(
-    [
-      "🔥 Cloudflare context bindings enabled for dev server",
-      "",
-      `Config path: ${configPath ? relative(".", configPath) : colorize("yellow", "cannot find `wrangler.toml`")}`,
-      `Persist dir: \`${relative(".", persistDir)}\` ${addedToGitIgnore ? colorize("green", "(added to `.gitignore`)") : ""}`,
-    ].join("\n"),
-  );
+  if (!nitro.options.cloudflareDev?.hideConsoleBox) {
```

Comment:
i think we can simply call it `silent: true` wdyt?

Reply:
Renamed